### DISCUSSION TITLE
Ability to start multiple searches at once or Manual to take precedence

### DISCUSSION THREAD

**Discussion by Rouzax** [pymedusa/Medusa#9558] [Ideas & Feature requests]
Would it be possible to add an option to allow an x number of searches at once?
Often when I want to do a manual search I can't because Medusa is running a search in the background.
![image](https://user-images.githubusercontent.com/4103090/118782793-c1ba2200-b88e-11eb-81bd-9bd9a974c480.png)

**Comment by p0psicles** [comment]
You can run the search. It's only queued when another backlog search is running. And this is pretty much by design. Running allot of threads parrallel will often result in an unresponsive system. Due to all the IO involved (network, disk)

**Reply by Rouzax** [reply]
Think I found out was happening,
I did a season search and it looks like it is not updating the `Queued search...` text even though in the background it is actually searching

**Reply by Rouzax** [reply]
I have plenty of resources on my system so in my case I would prefer the ability to start a search of an episode to multiple providers at the same time, instead of walking through them serial

**Comment by Rouzax** [comment]
@p0psicles, it appears there is a bug here as well.
The state of individual episode searches is changed but not for a season pack search, that always stays on `Queued search...`

**Reply by p0psicles** [reply]
Yeah I know. I should take a look at that

**Reply by Rouzax** [reply]
Want me to open up a real issue to keep track of it?

**Reply by p0psicles** [reply]
Sure
